src/components/forms/AddressInput/index.tsx

```diff
@@ -66,7 +68,17 @@ const AddressInput = ({
             logError(Errors._101, err.message)
           }
         } else {
-          fieldMutator(address)
+          // A regular address hash
+          let checkedAddress = address
```

Comment:
```suggestion
          let checkedAddress = address.toLowerCase()
```

Reply:
The last chance for good UX.

Reply:
I suspect most people will have either normalized (checksummed) addresses, or all-lowercase.
If a person makes a mistake however, or pastes someone else's mistake, this will help them spot it early.

Anyway, up to @tschubotz.

Reply:
> If a person makes a mistake however, or pastes someone else's mistake, this will help them spot it early.

My point is that people mostly don't care about checksum when entering an address and figuring out what the mistake is may be hard for them

Reply:
It will tell them "must be a valid address", so they'll probably go and double-check or copy&paste it again.
That's the whole point of the checksum.

Reply:
> The last chance for good UX.

😂 

> I suspect most people will have either normalized (checksummed) addresses, or all-lowercase.
> If a person makes a mistake however, or pastes someone else's mistake, this will help them spot it early.

I agree with this. And also with Ivan's latest comment.

Reply:
I think most of the users think about it as a case-insensitive login such as email or a phone number. I don't think this checksum things brings any security/ux benefits

Reply:
Maybe we can ask Kristina